Jenkinsfile

```diff
@@ -55,8 +54,8 @@ for (int i = 0; i < platforms.size(); ++i) {
 /* Execute our platforms in parallel */
 parallel(branches)
 
-
-stage('Verify demos')
+// TODO Restore once JENKINS-56632 is done
+/*stage('Verify demos')
```

Comment:
Needs to be reverted

Reply:
@oleg-nenashev since it causes sometimes some workspace collisions, I decided commented. #101 is reverting the comment, although is in progress for the moment.